tests/hirte_tests/provision/containers.py

```diff
@@ -3,23 +3,27 @@
 import io
 import os
 import tarfile
+from typing import Any, Iterator, Optional, Tuple, Union, IO
```

Comment:
Shouldn't there be an empty line between `import ...` and `from ...` imports?

Reply:
Yes, added it. Thanks!